unitary/quantum_chess/circuit_transformer.py

```diff
@@ -400,48 +400,43 @@ def transform(self, circuit: cirq.Circuit) -> cirq.Circuit:
         return cirq.Circuit(updater.add_swaps())
 
 
-class SycamoreDecomposer(cirq.PointOptimizer):
-    """Optimizer that decomposes all three qubit operations into
+def transform_controlled_iswap(op: cirq.Operation, index: int) -> cirq.OP_TREE:
```

Comment:
The implementation of `controlled_iswap` decompositions also use deprecated optimizers -- will these be updated in a follow-up PR? 

https://github.com/quantumlib/unitary/blob/5d7e70966dd0061c3473bb29984a38a3cb32c943/unitary/quantum_chess/controlled_iswap.py#L119


Also, `cirq.optimize_for_target_gateset(circuit, gateset=cirq.SqrtIswapTargetGateset(), ignore_failures=False)` is able to convert the controlled iswap cases as well (see the [colab](https://colab.sandbox.google.com/drive/1sjacBgm1oqa2giTQ_QUlu_GxKqJ9GLsR?resourcekey=0-UIKWzHrpTw58dWE-VPnqlw&usp=sharing#scrollTo=E3sT1gth8xJR)). My guess is that the custom decompositions used below are more efficient than the default ones?